app/src/components/dialog/DialogVisual.tsx

```diff
@@ -0,0 +1,59 @@
+import Avatar from '@material-ui/core/Avatar';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { mdiAlert, mdiCheck, mdiExclamation, mdiFileCheckOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import clsx from 'clsx';
+import React from 'react';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  dialogVisual: {
+    padding: 24,
+    boxSizing: 'content-box',
+    margin: '48px auto 0'
+  },
+  error: {
+    color: '#a12622',
+    backgroundColor: '#F5E9E8'
+  },
+  warning: {
+    color: '#a12622',
+    backgroundColor: '#a12622 10%'
+  },
+  info: {
+    color: '#0972d3',
+    backgroundColor: '#F2F8FD'
+  },
+  success: {
+    color: '#2d4821',
+    backgroundColor: '#dff0d8'
+  }
+}));
+
+type Severity = 'error' | 'info' | 'success' | 'warning';
+
+const defaultIcons: Record<Severity, string> = {
+  error: mdiExclamation,
+  info: mdiFileCheckOutline,
+  success: mdiCheck,
+  warning: mdiAlert
+};
+
+interface IDialogVisualProps {
+  severity?: Severity;
+  icon?: string;
+}
+
+const DialogVisual = (props: IDialogVisualProps) => {
```

Comment:
Is this used anywhere (or going to be later)?

Reply:
Was used for a demo but got removed. I strongly suspect it will be used later, though